app/models/user.rb

```diff
@@ -474,6 +476,18 @@ def managed_active_teenagers_count
     User.active_teenager.joins(organizer_positions: :event).where(events: { id: managed_events }).distinct.count
   end
 
+  def linked_discord_account?
+    discord_id.present?
+  end
```

Comment:
Rename it's usages too
```suggestion
  def has_discord_account?
    discord_id.present?
  end
```

Reply:
[`cf97b36` (#11791)](https://github.com/hackclub/hcb/pull/11791/commits/cf97b36c41887cc6ba22284da3bbfd642bbe177d)